lib/datasource/adoptium-java/readme.md

```diff
@@ -1,5 +1,8 @@
 This datasource returns releases from [Adoptium](https://adoptium.net/) API.
 
-It uses `project=jdk&release_type=ga&sort_method=DATE&sort_order=DESC&vendor=adoptium` as filter parameters.
+It uses `image_type=<jre|jdk>&project=jdk&release_type=ga&sort_method=DATE&sort_order=DESC&vendor=adoptium` as filter parameters.
 
 It only uses the first 50 pages with 50 items per page.
+
+Use `java-jdk` or `java` as `lookupName` to get releases with JDK available or use `java-jre` to only get releases with JRE available.
+Currently only the LTS releases have a JRE available.
```

Comment:
```suggestion
Currently only the LTS releases of Java come with the JRE.
```

I think we mean that only LTS releases of Java come with the Java Runtime Environment?